website/source/docs/providers/aws/r/route.html.markdown

```diff
@@ -35,12 +35,14 @@ The following arguments are supported:
 * `destination_cidr_block` - (Required) The destination CIDR block.
 * `vpc_peering_connection_id` - (Optional) An ID of a VPC peering connection.
 * `gateway_id` - (Optional) An ID of a VPC internet gateway or a virtual private gateway.
```

Comment:
Since the `gateway_id` field is already overloaded to support both `internet gateway or a virtual private gateway`, would it make sense to also add the NAT Gateway id to that field and not define the new `nat_gateway_id`? I'm not familiar with the conventions here at all, but figured I would ask the question.